inspectit-oce-documentation/src/docs/asciidoc/_includes/configuration/open-census-configuration.adoc

```diff
@@ -0,0 +1,27 @@
+=== Using OpenCensus Library with inspectIT OCE
+
+If you plan to use the OpenCensus library in an application which will, later on, be instrumented with inspectIT OCE, some special rules do apply.
+Following these rules will make sure that there are no run-time problems in your application.
+Furthermore, a correct configuration will make it possible to combine metrics and traces that you manually collect using the OpenCensus instrumentation library with the ones collected by the inspectIT OCE agent.
+
+. Make sure you are using the same version of OpenCensus as inspectIT OCE
++
+The inspectIT OCE agent in version {inspectit-oce-version} internally uses OpenCensus in version {opencensus-version}.
+Please adapt any OpenCensus dependency in your application to this version to avoid run-time conflicts.
++
+```XML
+<dependency>
+    <groupId>io.opencensus</groupId>
+    <artifactId>opencensus-api</artifactId>
+    <version>0.18.0</version>
+</dependency>
+```
+. Set the `inspectit.publish-opencensus-to-bootstrap` property to `true`
++
+Setting the above property to `true` is telling inspectIT OCE that you plan to use the OpenCensus library in combination to the agent.
+The inspectIT OCE agent will make sure that all OpenCensus classes are then loaded by the bootstrap class loader.
+This ensures that OpenCensus implementation is shared between your manual instrumentation and the agent instrumentation, making the combing of data possible.
+. Add agent to the start of JVM
++
+You can only install the agent at the <<Adding the Agent to the JVM,start of the JVM>>.
+You can not attach the agent during the run-time, as at this point it's already too late for the inspectIT OCE agent to perform needed setup described in the previous point.
```

Comment:
You can acutally attach it, but the metrics/traces won't be combined. And when attaching later on it is safest to disable the pushing to the bootstrap.

Reply:
True, was also asking my self if we need to document this.. Actually, it would still be possible to combine if you push all the dependencies to the bootstrap your self.. Maybe this is too complicated or advanced topic somehow? Maybe for the FAQ section?

Reply:
I would leave out the manual pushing to the bootstrap for now.
I would just say that your data wont be combined if you attach it later to the JVM.